views/index.pug

```diff
@@ -11,30 +11,38 @@ html
     meta(http-equiv="Content-Type", content="text/html; charset=utf-8")
     meta(name="viewport", content="width=device-width, initial-scale=1")
     title #{__('TITLE', community)}
+    link(href="//static.uqcs.org/bulma/main.css" rel="stylesheet" type="text/css")
     link(href="css/style.css", rel="stylesheet", type="text/css")
-    link(href="//fonts.googleapis.com/css?family=Lato:300,400,700,900,700italic|Open+Sans:700italic,400,600,300,700,800", rel="stylesheet", type="text/css")
   body
-    #wrapper
-      .main
-        .header
-          img(src="images/logo.svg", onerror="this.src='images/logo.svg.png'", class="logo")
-          h2 #{__('HEADER', community)}
-        .content
-          .information
+    section(class="section")
+      .container
+        .columns.is-centered.is-mobile
+          .column#body
+            .my-1#header
+              img(src="images/logo.svg", onerror="this.src='images/logo.svg.png'", id="logo")
+              h4(class="title is-4 my-5") #{__('HEADER', community)}
             form(method="POST", action="invite")#join-form.form
-              input(type="email", name="email", autofocus, placeholder=__('ENTER_EMAIL'))#slack-email.field
-              if tokenRequired
-                input(type="text", name="token", placeholder=__('ENTER_TOKEN'))#slack-token.field
+              .field
+                label(class="label" for="slack-email") Email Address
+                .control
+                  input(class="input", type="email", name="email", required, autofocus, placeholder="a.smith@example.com")#slack-email.field
+                p(class="help") Valid UQ student email formats are: 
+                  span first.last@uqconnect.edu.au, 
+                  span first.last@uq.net.au, 
+                  span s1234567@student.uq.edu.au. 
               if !!recaptchaSiteKey
-                div(class="g-recaptcha", data-sitekey=recaptchaSiteKey)
-              input(type="submit", value="Join").submit
-        .content
-          .information
-            p Already a slack user? 
+                .field
+                  .control
+                    div(class="g-recaptcha", data-sitekey=recaptchaSiteKey)
+              .forminformation
+              
+              .buttons.wide.mb-1
+                input(class="button is-link" type="submit", value="Join").submit
+            p(class="help") By joining the UQCS Slack, you agree to have read and understood the 
+              a(href="https://uqcs.org/code-of-conduct" target="_blank") Code of Conduct
+            p(class="help") Already a slack user? 
               a(href=`https://${slackUrl}`) Sign in to #{community} here
```

Comment:
Personally, I would like to see this be a button or at least alongside the Join button.

Reply:
IMO, having a button to sign in that is next to the join button in a UX sense, does not make sense here. As this is a form, by moving the login button next to the join button you imply that the user must fill out the form in order to sign into the Slack which is not the case.

When separating it as a link at the bottom, you imply to the user that to login to the Slack you need to navigate to a different website.

Reply:
I think it makes sense. Sign up forms commonly offer a log in utton (e.g. Google login) alongside the next button. My problem with the link is that it's very hard to notice, especially now there's the CoC link as well. 

Ideally, we should make it more obvious that returning users can go straight to https://uqcs.slack.com. Even if it's not a dedicated button, the text could be bigger (e.g. not .help and just normal size).